gdpr_assist/admin/tool.py

```diff
@@ -1,6 +1,7 @@
 """
 Personal data admin tool
 """
+from __future__ import unicode_literals
```

Comment:
We should probably do a separate sweep to add future statements all over the place and make the library python 3 compatible.